models/rubicon/arbitrum/rubicon_arbitrum_trades_legacy.sql

```diff
@@ -0,0 +1,11 @@
+{{ config( 
+  schema = 'rubicon_arbitrum',
+  alias = alias('table_name', legacy_model=True),
```

Comment:
interesting, this one didn't have a legacy file?

Reply:
I think it was added recently.